app/components/ConnectDapp/ApproveConnection/ApproveConnection.jsx

```diff
@@ -66,19 +66,13 @@ const ApproveConnection = ({
     }
   }
 
-  const rejectSession = async (error?: string) => {
```

Comment:
This function was never intentionally being invoked in this component with the error param 👀

Reply:
I checked the `dev` branch and the reported error is not happening anymore. I think this PR is safe to be merged if you indeed checked that no one is using the `error` param.